src/briefcase/bootstraps/pyside6.py

```diff
@@ -79,34 +79,25 @@ def pyproject_table_linux_system_debian(self):
 
 system_runtime_requires = [
     # Derived from https://doc.qt.io/qt-6/linux-requirements.html
+    "libxext6",
     "libxrender1",
-    "libxcb-render0",
-    "libxcb-render-util0",
+    "libx11-xcb1",
+    "libxkbcommon-x11-0",
+    "libxcb-image0",
+    "libxcb-cursor0",
     "libxcb-shape0",
     "libxcb-randr0",
     "libxcb-xfixes0",
-    "libxcb-xkb1",
     "libxcb-sync1",
-    "libxcb-shm0",
     "libxcb-icccm4",
     "libxcb-keysyms1",
-    "libxcb-image0",
-    "libxcb-util1",
-    "libxkbcommon0",
-    "libxkbcommon-x11-0",
     "libfontconfig1",
-    "libfreetype6",
-    "libxext6",
-    "libx11-6",
-    "libxcb1",
-    "libx11-xcb1",
     "libsm6",
     "libice6",
     "libglib2.0-0",
     "libgl1",
-    "libegl1-mesa",
+    "libegl1",
     "libdbus-1-3",
-    "libgssapi-krb5-2",
 ]
 """
```

Comment:
I went through the effort of de-duping this a bit. Several of these are implicitly included as dependencies by others and at least one was a dummy package whose usefulness expired long ago. I also got rid of the kerberos package....I htink that was just something about my machine.